reanahub/reana-commons
# rabbitmq: make queues durable for persistence

closes https://github.com/reanahub/reana/issues/500



Testing Instructions without PR:

1. `max parallel running  workflows helm variable` -> 1
2. Launch 3 workflows.
3. While the 1st is running and 2nd and 3rd are in the queue, kill the queue pod (`message-broker`)
4. The Pod will be restarting again and check the queue and workflows are finished.

Testing Instructions with PR:

1. `max parallel running  workflows helm variable` -> 1
2. Rebuild the message broker `reana-dev docker-build -c reana-message-broke`
3. Follow the step 3 and 4 from above.
